### PR TITLE
Use Foreman GH Actions

### PR DESCRIPTION
**Pull request by chris1984**

### CODE REVIEW

**Review by ofedoren — APPROVED** [review]
Thanks, @chris1984, LGTM.

I'd also like to ask you if you could update the `data` for the tests and maybe add GHA for running the test suite as well?

File: hammer_cli_foreman_virt_who_configure.gemspec [hunk 1]
```diff
@@ -12,19 +11,29 @@ Gem::Specification.new do |s|
...
-    locales.map { |loc| "#{loc}/LC_MESSAGES/hammer-cli-foreman-virt-who-configure.mo" }
+  locales.map { |loc| "#{loc}/LC_MESSAGES/hammer-cli-foreman-virt-who-configure.mo" }
 
   s.test_files       = Dir['{test}/**/*']
   s.extra_rdoc_files = Dir['{doc}/**/*', 'README*']
   s.require_paths = ["lib"]
+  s.required_ruby_version = '>= 2.7', '< 3.2'
+
+  s.add_dependency 'hammer_cli', '~> 3.10', '>= 3.10.0'
+  s.add_dependency 'hammer_cli_foreman', '~> 3.9'
```
Comment (ofedoren) [inline hammer_cli_foreman_virt_who_configure.gemspec]: I'd rather pin for major version change, i.e. `< 4.0` for both...

File: test/functional/config/config_list_test.rb [hunk 2]
```diff
@@ -34,7 +34,7 @@
       api_expects(:configs, :index).with_params(
         'page' => '2',
```
Comment (ofedoren) [inline test/functional/config/config_list_test.rb]: Make this a number instead of string and the test should pass.

**Review by ofedoren — CHANGES_REQUESTED** [review]
Thanks, @chris1984 ! LGTM, just some small things inline.

**Comment by chris1984** [comment]
@ofedoren updated. The Ruby 2.7 tests are failing with all kinds of fun non initialized errors, I am wondering if we should just pull Ruby 2.7 out or just leave it as for now?

**Comment by ofedoren** [comment]
@chris1984 Ruby 2.7 failures due to warnings can be fixed via https://github.com/theforeman/hammer-cli/blob/master/Rakefile#L14. Let's do that and it's ready to go :)

**Pull request merged**